### ISSUE TITLE
known-properties with -webkit-appearance property

### ISSUE THREAD

**Issue by MisterMashu** [CSSLint/csslint#693]
The following css:

```css
input[type=search]{
    -webkit-appearance: textfield;
}
```

errs out with the following on my csslint:

```
Error - Expected (none | button | button-bevel | caps-lock-indicator | caret | checkbox | default-button | listbox	| listitem | media-fullscreen-button | media-mute-button | media-play-button | media-seek-back-button	| media-seek-forward-button	| media-slider | media-sliderthumb | menulist	| menulist-button	| menulist-text	| menulist-textfield | push-button	| radio	| searchfield	| searchfield-cancel-button	| searchfield-decoration | searchfield-results-button | searchfield-results-decoration | slider-horizontal | slider-vertical | sliderthumb-horizontal | sliderthumb-vertical	| square-button	| textarea	| textfield	| scrollbarbutton-down | scrollbarbutton-left | scrollbarbutton-right | scrollbarbutton-up | scrollbargripper-horizontal | scrollbargripper-vertical | scrollbarthumb-horizontal | scrollbarthumb-vertical | scrollbartrack-horizontal | scrollbartrack-vertical) but found 'textfield'. (known-properties)
```

and is totally ignored on csslint.net. you can type in "bananas" instead of "textfield" and it does not throw a warning, even with known-properties checked.

Either way, the error message I'm getting is weird because "textfield" is one of the listed options, yet it thinks it's invalid.

**Comment by nschonni** [comment]
Think this might actually be related to https://github.com/CSSLint/parser-lib/pull/225
Can you try editing out the tabs in your local copy to see if that resolves the issue?

**Comment by diegoteliz** [comment]
I had exactly the same problem and after replacing tabs by spaces on `Properties.js` file it worked properly.

Thanks @nschonni 👍
